plugin_build/README.md

```diff
@@ -0,0 +1,162 @@
+# GRPC Proxy How To
```

Comment:
I understand `plugin_build` is just a guide. Is this going to move to the docs?

Reply:
Eventually, this is alpha, so keeping a readme here is a good idea until we formalize